code/_onclick/hud/fullscreen.dm

```diff
@@ -156,9 +156,33 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/fullscreen)
 	icon_state = "noise"
 
 /atom/movable/screen/fullscreen/high
-	icon = 'icons/hud/screen_gen.dmi'
-	screen_loc = "WEST,SOUTH to EAST,NORTH"
 	icon_state = "druggy"
+	alpha = 255
+	plane = LIGHTING_PLANE
+	layer = LIGHTING_ABOVE_ALL + 1 //Infinity plus one (not actually)
+	blend_mode = BLEND_MULTIPLY
+
+/atom/movable/screen/fullscreen/high/update_for_view(client_view)
+
+	animate(src, flags = ANIMATION_END_NOW) //Stop all animations.
+
+	. = ..()
+
+	color = list(1,0,0,0, 0,1,0,0, 0,0,1,0, 0,0,0,1, 0,0,0,0) //We convert it early to avoid a sudden weird jitter.
+	alpha = 0
+
+	animate(src, alpha = 255, time = 5 SECONDS) //Fade in.
+
+	addtimer(CALLBACK(src, PROC_REF(start_hue_rotation)), 5 SECONDS)
+
+/atom/movable/screen/fullscreen/high/proc/start_hue_rotation()
+	animate(src, color = color_matrix_rotate_hue(1), loop = -1, time = 2 SECONDS) //Start the loop.
+	var/step_precision = 18 //Larger is more precise rotations.
+	for(var/current_step=1,current_step<=step_precision-1,current_step++) //We do the -1 here because 360 == 0 when it comes to angles.
```

Comment:
Also you resolved my other review without any comment, you should go back to it

Reply:
It auto-resolved due to edits.